assets/js/components/ecosystems/FooterSection.js

```diff
@@ -4,34 +4,36 @@ export default class Footer extends Component {
   render() {
     return (
       <div className="Footer">
-        <div className="summary flexMiddie">
-          <div className="siteMap">
+        <div className="siteMap">
 
-            <ul>
-              <li><a href="/">Home</a></li>
-              <li><a href="/opensource">Open Source</a></li>
-              <li><a href="/contact">Contact Us</a></li>
-            </ul>
+          <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/opensource">Open Source</a></li>
+            <li><a href="/contact">Contact Us</a></li>
+          </ul>
 
-            <ul>
-              <li><a href="/services/node-js-development">Backend</a></li>
-              <li><a href="/services/front-end-development">Frontend</a></li>
-              <li><a href="/services/design">Design</a></li>
-            </ul>
-          </div>
-          <div className="contact-info">
-            <h2><i className={'icon-mail'} /><a href="mailto:hello@langa.io">hello@langa.io</a></h2>
```

Comment:
The items under contact information should not be wrapped in h2 tags.  Something like:
```html
<section>
    <h2>Contact Information</h2>
    <ul>
        <li><a href="#"><i className="icon-mail" /> Email</a></li>
        <li><a href="#"><i className="icon-phone" />(757) 828-7656</a></li>
    </ul>
</section>
```
would be more correct.